xcodeproj/internal/top_level_targets.bzl

```diff
@@ -235,6 +235,15 @@ def process_top_level_target(
             ),
         )
 
+    if automatic_target_info.strings:
+        additional_files.extend(
+            getattr(
+                ctx.rule.files,
+                automatic_target_info.strings,
+                [],
+            ),
+        )
+
```

Comment:
I believe strings are handled by the resource management. I think all of the handling can be removed, and we only need the example. You can confirm in the BwX example at compile and runtime?

Reply:
![Screen Shot 2022-09-27 at 4 59 19 PM](https://user-images.githubusercontent.com/566328/192634286-52d76d73-75d5-4bbe-b6fb-828dda74bc93.png)

Seeing this after I remove all the handling logic and just leave in the edits to the example

Reply:
Yep, that is just files in the `extra_files`, you should be able to see them still in the `resources` sections.

Reply:
![Screen Shot 2022-09-27 at 5 18 05 PM](https://user-images.githubusercontent.com/566328/192637351-72996e38-e6da-45fd-8af6-d8a3d6a24359.png)

You are right! Missed those at first glance